op-service/eth/account_proof.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 type StorageProofEntry struct {
-	Key   hexutil.Big     `json:"key"`
+	Key   hexutil.Bytes   `json:"key"`
```

Comment:
I think we've picked up this change from the other PR. Should sort itself out when that gets merged and we can rebase though.